.github/workflows/sync-testnets.yml

```diff
@@ -37,16 +45,34 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
+      - name: Installing requirements
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y make build-essential jq screen lshw dmidecode fio
+
+      - name: Install Docker
+        run: |
+          sudo apt update
+          sudo apt install apt-transport-https ca-certificates curl software-properties-common -y
+          curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+          sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+          sudo apt update
+          sudo apt install docker-ce -y
+
       - name: Build docker image
         run: docker buildx build --platform=linux/amd64 -t current_branch_image -f Dockerfile --build-arg COMMIT_HASH=${{ steps.settings.outputs.COMMIT_HASH }} --build-arg BUILD_TIMESTAMP=${{ steps.settings.outputs.BUILD_TIMESTAMP}} --load .
 
       - name: Setup Go environment
         uses: actions/setup-go@v4.0.0
+        with:
+          go-version: '1.20'
+          check-latest: true
+          cache: true
 
       - name: Install Sedge environment
         run: |
           echo "Downloading sedge sources..."
-          git clone https://github.com/NethermindEth/sedge.git sedge --branch main --single-branch
+          git clone https://github.com/NethermindEth/sedge.git sedge --branch develop --single-branch
```

Comment:
holesky is only available in develop of sedge as hack added by marcus ryt ?

Reply:
@shashankshampi Yes but will be soon on main - decided to switch to develop to have a possibility to reach new features sooner